pkg/test/controller/reconcile.go

```diff
@@ -63,6 +66,28 @@ func StartTestManagerInstance(env *envtest.Environment, testType test.Type, whCf
 	return mgr, stopFunc
 }
 
+// Without the scheme, dynamic test returns `no kind is registered for the type
+// XXX in scheme` error.
+// Even with the scheme, dynamic test still returns `the server could not find
```

Comment:
Odd - I'm guessing the CCC CRD isn't registered, as it looks like you handle the case where the CCC object does not exist.   Do we know where the error is coming from / if the CRD is registered?

Reply:
I debugged for while today and finally realized the CRD installation and scheme registration need to be done separately. With the CRD, unstructured can be created/retrieved; with the scheme, the typed object can be created/retrieved.